apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -558,6 +573,67 @@ export const SignInMethodCustomization: FunctionComponent<SignInMethodCustomizat
         );
     };
 
+    const SMSOTPConflictMessage = () => {
+
+        const { idpList } = smsValidationResult;
+
+        return (
+            <Message
+                data-componentid="jit-provisioning-mfa-in-sequence-warning-message"
+                data-testid="jit-provisioning-mfa-in-sequence-warning-message"
+                type="info"
+                // Semantic hides warning messages inside <form> by default.
+                // Overriding the behaviour here to make sure it renders properly.
+                content={ (
+                    <>
+                        {
+                            idpList.length > 1
+                                ? (
+                                    <>
+                                        <Trans
+                                            i18nKey={
+                                                "console:develop.features.applications.edit.sections." +
+                                                "signOnMethod.sections.authenticationFlow.sections." +
+                                                "stepBased.federatedSMSOTPConflictNote.multipleIdps"
+                                            }>
+                                            Asgardeo requires the user&apos;s profile containing the
```

Comment:
The reason for me to use `&apos;s` instead of `'` is that following error  is thrown when `'` is used->  

> 599:71  error  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities